src/test/java/io/lettuce/core/commands/reactive/HashReactiveCommandIntegrationTests.java

```diff
@@ -49,4 +49,10 @@ public void hgetallStreaming() {
 
     }
 
+    @Test
+    @Disabled("API differences")
+    public void hexpiretime() {
```

Comment:
nit: needs some cleanup

Reply:
I am currently checking these up
For some reason in cluster mode the command fails to parse the response